src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -199,10 +203,7 @@ def _get_job_ids(self, req: JobRequest) -> List[str]:
         if req.has_batch_id():
             return self._jm.update_batch_job(req.batch_id)
 
-        try:
-            return req.job_id_list
-        except Exception as ex:
-            raise JobRequestException(ONE_INPUT_TYPE_ONLY_ERR) from ex
+        return req.job_id_list
```

Comment:
Because `req.job_id_list` should only reasonably throw one kind of exception, and that's already `JobRequestException(ONE_INPUT_TYPE_ONLY_ERR)`? Right ..?

Reply:
Yes, that's right... I am confused as to how that other code got in. Must have been a merge mishap. Anyway, it's fixed now.